Source/UI/IdentityUI.swift

```diff
@@ -753,3 +774,45 @@ private final class DismissableNavigationController: UINavigationController {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+private final class SharedWebCredentialsController: NSObject {
+    // Stored anonymously to retain the ASAuthorizationController
+    private var authorizationController: NSObject?
+
+    var didCompleteWithAuthorization: (String, String) -> Void = { _, _ in }
+    var didCompleteWithError: (Swift.Error) -> Void = { _ in }
+
+    override init() {
```

Comment:
Can we provide the result as a `Swift.Result` and add closure to the initialiser?